shardingsphere-kernel/shardingsphere-authority/shardingsphere-authority-core/src/test/java/org/apache/shardingsphere/authority/provider/schema/DatabasePermittedPrivilegesProviderAlgorithmTest.java

```diff
@@ -41,6 +42,7 @@ public void assertBuildAuthorityRegistry() {
         Optional<ShardingSpherePrivileges> privileges = actual.findPrivileges(new Grantee("user1", "127.0.0.2"));
         assertTrue(privileges.isPresent());
         assertTrue(privileges.get().hasPrivileges("test"));
+        assertTrue(privileges.get().hasPrivileges(new DatabaseAccessSubject("*"), Collections.emptyList()));
```

Comment:
The param of DatabaseAccessSubject should be a name of logic database, like `db_dal_admin` in the `PROP_USER_DATABASE_MAPPINGS`, can not be `*`, is it?